Face/Verify/Program.cs

```diff
@@ -0,0 +1,289 @@
+            await Client.PersonGroup.TrainAsync(GroupId);
+
+            // Wait until the training is completed.
+            while (true)
+            {
+                await Task.Delay(1000);
+                var status = await Client.PersonGroup.GetTrainingStatusAsync(GroupId);
+                Console.WriteLine($"Response: Success. Group \"{GroupId}' training process is {status.Status}");
+                if (status.Status != TrainingStatusType.Running)
+                {
+                    break;
+                }
+            }
+
+            List<Guid> faceIds = new List<Guid>();
+
+            // Read image file. 
+            using (FileStream stream = new FileStream(Path.Combine("Images", SourceImageFileName1), FileMode.Open))
+            {
+                // Detect faces from image stream. 
+                IList<DetectedFace> detectedFaces = await Client.Face.DetectWithStreamAsync(stream);
+                if (detectedFaces == null || detectedFaces.Count == 0)
+                {
+                    Console.WriteLine($"[Error] No face detected from image `{SourceImageFileName1}`.");
+                    return;
+                }
+
+                Console.WriteLine($"{detectedFaces.Count} faces detected from image `{SourceImageFileName1}`.");
+                if (detectedFaces[0].FaceId == null)
+                {
+                    Console.WriteLine("[Error] Parameter `returnFaceId` of `DetectWithStreamAsync` must be set to `true` (by default) for Identify purpose.");
+                    return;
+                }
+
+                // Add detected faceId to faceIds.
+                faceIds.Add(detectedFaces[0].FaceId.Value);
+            }
+
+            // Verification example for faces of the same person.
+            VerifyResult VerifyResults = await Client.Face.VerifyFaceToPersonAsync(faceIds[0], p.PersonId, GroupId);
+            Console.WriteLine(
+                VerifyResults.IsIdentical
+                    ? $"Faces from {SourceImageFileName1} & {p.Name} are of the same (Positive) person, similarity confidence: {VerifyResults.Confidence}."
+                    : $"Faces from {SourceImageFileName1} & {p.Name} are of different (Negative) persons, similarity confidence: {VerifyResults.Confidence}.");
+
+            // Delete the person group.
+            Console.WriteLine($"{Environment.NewLine}Delete the person group ({GroupId})");
+            await Client.PersonGroup.DeleteAsync(GroupId);
+        }
+
+        public static async Task Verify_in_LargePersonGroup()
+        {
+            // Create a large person group.
+            string GroupId = Guid.NewGuid().ToString();
+            Console.WriteLine($"Create a large person group ({GroupId}) {Environment.NewLine}");
+            await Client.LargePersonGroup.CreateAsync(GroupId, GroupId);
+
+            // Create a large person group person.
+            Person p = new Person();
+            p.Name = "Dad";
+            p.UserData = "Person for sample";
```

Comment:
`Person p = new Person { Name = "Dad", UserData = "Person for sample" };`

Reply:
done